Orange/tests/test_table.py

```diff
@@ -53,285 +54,146 @@ def test_filename(self):
         self.assertTrue(d.__file__.endswith("test2.tab"))  # platform dependent
 
     def test_indexing(self):
-        import warnings
-
-        with warnings.catch_warnings():
-            warnings.simplefilter("ignore")
+        with catch_warnings():
+            simplefilter("ignore")
```

Comment:
I'd actually prefer having just `warnings` imported at the top and these used like `warnings.simplefilter('ignore')`